skyportal/handlers/api/group.py

```diff
@@ -358,3 +379,113 @@ def delete(self, group_id, username):
+            application/json:
+              schema:
+                type: object
+                properties:
+                  stream_id:
+                    type: integer
+                required:
+                  - stream_id
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Success'
+                    - type: object
+                      properties:
+                        data:
+                          type: object
+                          properties:
+                            group_id:
+                              type: integer
+                              description: Group ID
+                            stream_id:
+                              type: integer
+                              description: Stream ID
+        """
+        data = self.get_json()
+        group_id = int(group_id)
+        stream_id = data.get('stream_id')
+        stream = Stream.query.filter(Stream.id == stream_id).first()
+        if stream is None:
+            return self.error(
+                "Specified stream_id does not exist."
+            )
+        else:
+            # Add new GroupStream
+            gs = GroupStream.query.filter(
+                GroupStream.group_id == group_id
+            ).filter(
+                GroupStream.stream_id == stream_id
+            ).first()
+            if gs is None:
+                DBSession.add(
+                    GroupStream(group_id=group_id, stream_id=stream_id)
+                )
+            else:
+                return self.error(
+                    "Specified stream is already associated with this group."
+                )
+        DBSession().commit()
+
+        self.push_all(action='skyportal/REFRESH_GROUP',
+                      payload={'group_id': group_id})
+        return self.success(data={'group_id': group_id, 'stream_id': stream_id})
+
+    @permissions(['System admin'])
+    def delete(self, group_id, stream_id):
+        """
+        ---
+        description: Delete an alert stream from group
+        parameters:
+          - in: path
+            name: group_id
+            required: true
+            schema:
+              type: integer
+          - in: path
+            name: stream_id
+            required: true
+            schema:
+              type: integer
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+        """
+        group = Group.query.get(group_id)
+        stream = Stream.query.filter(Stream.id == int(stream_id)).first()
+        stream_id = stream.id
+        if stream is None:
+            return self.error(
+                "Specified stream_id does not exist."
+            )
+        else:
+            if group.single_user_group:
+                return self.error("Cannot delete streams from single user groups.")
```

Comment:
We should support this, for when people change institutions.

Reply:
True, removed.